frontend/templates/views/partials/content.j2

```diff
@@ -36,6 +36,23 @@
   {% include '/views/partials/level-toggle.j2' %}
   {% endif %}
 
+  {# Render widget to toggle experiments if there are any#}
```

Comment:
do we need to import amp-iframe here?

Reply:
There was no error, but I imported it anyways now.